docusaurus/docs/reference/ha/overview.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_label: Overview
+sidebar_position: 05
+---
+
+# Controller HA 
+
+## Overview
+
+Ziti controllers can be run in a cluster for high availablity and performance scaling.
```

Comment:
this needs an actual "overview" of what it even means to be HA. I like the warning, it just feels like extra words are necessary describing what this means. not so much 'why', just 'what'